packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -24,14 +25,27 @@ const disabledInstrumentations = new Set(
 )
 
 // Check for DD_TRACE_<INTEGRATION>_ENABLED environment variables
+const enabledIntegrationsOverrides = new Set()
 for (const [key, value] of Object.entries(process.env)) {
   const match = key.match(/^DD_TRACE_(.+)_ENABLED$/)
```

Comment:
This pattern is a little unfortunate. For example, to enable the `@prisma/client` library, we'd need an env var named `DD_TRACE_@PRISMA/CLIENT_ENABLED`.

Reply:
The configuration uses the integration name. Assuming the prisma integration name used is `prisma`, the `disabledByDefaultIntegrations `(mapping of integration name to package name) would equal `{ prisma: @prisma/client }`. So the configuration to enable it would be `DD_TRACE_PRISMA_ENABLED`

Reply:
Oh, I misunderstood. The disabled by default list is essentially a list of aliases. Got it.